fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -48,6 +48,18 @@ def self.ci?
       return false
     end
 
+    def self.unix?
+      !windows?
+    end
+
+    def self.windows?
+      (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM) != nil
+    end
+
+    def self.linux?
+      unix? and !mac?
```

Comment:
Let's not define linux by it being a unix while not being a mac, I'm sure some people would get a heart attack reading this 😅 

I just checked, `RUBY_PLATFORM` is `<arch>-linux` on linux. Maybe a check like that would be more appropriate.